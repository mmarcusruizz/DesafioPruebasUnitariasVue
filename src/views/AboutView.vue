<template>
    <div>
      <h1>About</h1>
      <p>Contenido de la p√°gina About</p>
    </div>
  </template>
  
  <script>
  export default {
    name: 'AboutView'
  }
  </script>