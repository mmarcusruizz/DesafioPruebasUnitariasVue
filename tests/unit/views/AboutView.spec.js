import { shallowMount } from '@vue/test-utils'
import AboutView from '@/views/AboutView.vue'

describe('AboutView', () => {
  it('renders the component', () => {
    const wrapper = shallowMount(AboutView)
    expect(wrapper.text()).toContain('About')
  }),
  it('matches the snapshot', () => {
    const wrapper = shallowMount(AboutView)
    expect(wrapper.html()).toMatchSnapshot()
  })

})