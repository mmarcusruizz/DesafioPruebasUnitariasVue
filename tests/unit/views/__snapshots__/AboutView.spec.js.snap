// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView matches the snapshot 1`] = `
<div>
  <h1>About</h1>
  <p>Contenido de la p√°gina About</p>
</div>
`;
